Scripts/generate_release_zip.sh

```diff
@@ -124,7 +124,7 @@ then
   echo "Run ObjC iOS tests ..."
   # iOS-App target runs Keychain-Accessing tests
   sh Scripts/xctest_crash_log.sh --delete-all
-  xcodebuild test \
+  xcodebuild clean test \
```

Comment:
Is the `clean` parameter needed for iOS but not macOS?

Reply:
I thought, code coverage data is only taken for iOS. The last ran test results from xcodebuild.